test/dynamo/test_export.py

```diff
@@ -4565,11 +4565,6 @@ def forward(self, x, b, y):
     return pytree.tree_unflatten([x], self._out_spec)""",  # NOQA: B950
         )
 
-        with self.assertRaisesRegex(
-            torch._dynamo.exc.Unsupported, "boolean masking setitem backwards"
-        ):
-            gm, _ = torch._dynamo.export(fn)(x, b, y)
```

Comment:
@sudoRicheek can we keep this line? Just remove the `with`

Reply:
Just curious, if there is no assert, why do we need this line?

Reply:
@sudoRicheek Good question. I guess it is good to show that it can run normally? Because if you remove both lines, the test is basically doing nothing.